lib/nerves_bootstrap.ex

```diff
@@ -77,6 +87,21 @@ defmodule Nerves.Bootstrap do
           """
         end
 
+    message =
+      if nerves_ver && Version.match?(nerves_ver, "< 1.8.0"),
```

Comment:
I think that the Nerves and Nerves.Bootstrap version checks should be separate. There's a chick-and-egg issue for the Nerves version since I don't see how you know what Nerves version is required until you get the version of Nerves.Bootstrap that cares. I don't think this matters, though.

Reply:
Right. This one evaluate until you get this new Nerves.Bootstrap version. So everything will run just fine until you `mix local.nerves` to update (typically after being prompted). From then on, you have this new Nerves.Bootstrap which would require the new Nerves as well and this check should run.